### PR TITLE
Add "join table" for visits

### PR DESCRIPTION
**Pull request by kifle23**
In this PR the following is done:

- [x] Create a table for vets, specializations, & visits
- [x] Insert data for vets, specializations, & visits
- [x] Write queries
The screenshot of the results is attached to this PR:

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -103,4 +103,77 @@ FROM owners o
...
+WHERE vt.name = 'William Tatcher'
+ORDER BY v.visit_date DESC
+LIMIT 1;
+
+--2--
+SELECT COUNT(DISTINCT v.animal_id)
+FROM visits v
+JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'Stephanie Mendez';
+
```
Comment (shella12) [inline queries.sql]: - [ ] The query 

```sql

SELECT COUNT(DISTINCT v.animal_id) FROM visits v JOIN vets vt ON vt.id = v.vet_id WHERE vt.name = 'Stephanie Mendez';
```

 is incorrect ❌ . 

The `COUNT(DISTINCT)` function should be used on the `animal_id `column, not the v column. For example the query can be written as following:

```sql

SELECT COUNT(DISTINCT a.id)
FROM visits v
JOIN animals a ON a.id = v.animal_id
JOIN vets vt ON vt.id = v.vet_id
WHERE vt.name = 'Stephanie Mendez';


```

This query will count the number of distinct animals that Stephanie Mendez has seen ✅ .  

The original query that you wrote would count the number of distinct visits that Stephanie Mendez has made, which is not what the question is asking for. However the question that the query is trying to answer is: How many different animals did Stephanie Mendez see?

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @kifle23, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Wrote all queries ✅ 
- All required data is inserted ✅ 
- All required tables are added ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (kifle23) [reply queries.sql]: Hi @shella12, thank you so much for your review and feedback. I have adjusted the query as per your request.
image

**Review by joel-kalema — APPROVED** [review]
Hi @kifle23 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights

The project is running successfully 👍 
 Good implementations👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**